### ISSUE TITLE
Squiz OperatorBracketSniff incorrectly reports and fixes operations using self::

### ISSUE THREAD

**Issue by yfedoruck** [squizlabs/PHP_CodeSniffer#615]
string to and after phpcbf --standard=Squiz :
```php
-        $cntPages = ceil( count($items) / self::ON_PAGE);
+        $cntPages = ceil((count($items) / ) self::ON_PAGE);
```
PHP Parse error:  syntax error, unexpected ')'

**Comment by aik099** [comment]
What is the error message on that line if you run `phpcs -s --standard=Squiz` on that file?

**Comment by gsherwood** [comment]
No error should have been reported with that line of code, so I've fixed that. I've also added another fix so that even if an error is reported, the brackets will be placed in the correct location. Thanks for reporting.

**Comment by stephenharris** [comment]
Just tested this, and can confirm this patch fixes the issue. Thank you!
